perceval/backends/core/confluence.py

```diff
@@ -91,6 +92,16 @@ def fetch(self, category=CATEGORY_HISTORICAL_CONTENT, from_date=DEFAULT_DATETIME
 
         return items
 
+    @staticmethod
+    def add_ancestors(url_prefix, item, content):
+        ancestors = []
+        if ('ancestors' in content) and (content['ancestors']):
+            for ancestor in content['ancestors']:
+                ancestors.append(url_prefix + ancestor['_links']['webui'])
```

Comment:
generally Perceval tries to have a one-to-one correspondence with the data in upstream, thus we avoid manipulating the data (something that is done by ELK during the data loading or enrichment)

Reply:
So you're basically proposing to remove `url_prefix +` from line 100 ?